eng/pipelines/templates/jobs/mgmt-release.yml

```diff
@@ -72,6 +78,26 @@ jobs:
             packagesToPush: '$(PIPELINE.WORKSPACE)/packages/**/*.nupkg;!$(PIPELINE.WORKSPACE)/packages/**/*.symbols.nupkg'
             nuGetFeedType: external
             publishFeedCredentials: Nuget.org
+      
+      - ${{ if eq(parameters.ShouldPublishToDevOps, 'true') }}:
+        - pwsh: |
+            if ('${{ parameters.DevOpsFeedID }}' -eq 'Private-DevOps-Feed'){
```

Comment:
We should use similar logic to https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/stages/archetype-net-release.yml#L263-L273 instead of letting folks choose.